CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ All notable changes to this project will be documented in this file.
 - [TypeScript] 'Cannot use 'in' operator to search for 'initializer'; decorator fix
 
 ### Changes
-
+- [Database] You can now update the random id schema by importing `import { setGenerator } from '@nozbe/watermelondb/utils/common/randomId'` and then calling `setGenerator(newGenenerator)`. This allows WatermelonDB to create specific IDs for example if your backend uses UUIDs.
```

Comment:
could you please move this to the top (under Unreleased)? Not your fault, just git didn't resolve this correctly after I released newest version :)